File: src/tutors.ts
export interface TutorProfile {
  id: string;
  name: string;
  displayName: string;
  email: string;
  assignedLevels: string[];
}

export const TUTOR_PROFILES: TutorProfile[] = [
  {
    id: "xiomara-villamizar",
    name: "Xiomara Villamizar",
    displayName: "Tutora Xiomara Villamizar",
    email: "xiomara@seroestar.com",
    assignedLevels: ["A1", "A2"]
  },
  {
    id: "ivoneth-frias",
    name: "Ivoneth Frias",
    displayName: "Tutora Ivoneth Frias",
    email: "ivoneth@seroestar.com",
    assignedLevels: ["A1", "A2"]
  },
  {
    id: "guerly",
    name: "Guerly",
    displayName: "Tutora Guerly",
    email: "guerly@seroestar.com",
    assignedLevels: ["A1", "A2"]
  },
  {
    id: "lashika",
    name: "Lashika",
    displayName: "Tutora Lashika",
    email: "lashika@seroestar.com",
    assignedLevels: ["A1", "A2"]
  }
];

export const DEFAULT_TUTOR = TUTOR_PROFILES[0];

export function findTutorByEmail(email?: string) {
  if (!email) return DEFAULT_TUTOR;
  return TUTOR_PROFILES.find((tutor) => tutor.email.toLowerCase() === email.toLowerCase()) || DEFAULT_TUTOR;
}

File: backend/tutors.py
TUTOR_PROFILES = [
    {
        "id": "xiomara-villamizar",
        "name": "Xiomara Villamizar",
        "display_name": "Tutora Xiomara Villamizar",
        "email": "xiomara@seroestar.com",
        "assigned_levels": ["A1", "A2"],
    },
    {
        "id": "ivoneth-frias",
        "name": "Ivoneth Frias",
        "display_name": "Tutora Ivoneth Frias",
        "email": "ivoneth@seroestar.com",
        "assigned_levels": ["A1", "A2"],
    },
    {
        "id": "guerly",
        "name": "Guerly",
        "display_name": "Tutora Guerly",
        "email": "guerly@seroestar.com",
        "assigned_levels": ["A1", "A2"],
    },
    {
        "id": "lashika",
        "name": "Lashika",
        "display_name": "Tutora Lashika",
        "email": "lashika@seroestar.com",
        "assigned_levels": ["A1", "A2"],
    },
]


def find_tutor_by_email(email):
    if not email:
        return TUTOR_PROFILES[0]
    return next((tutor for tutor in TUTOR_PROFILES if tutor["email"].lower() == email.lower()), TUTOR_PROFILES[0])


def find_tutor_by_name(name):
    if not name:
        return TUTOR_PROFILES[0]
    normalized = name.lower()
    return next(
        (
            tutor
            for tutor in TUTOR_PROFILES
            if tutor["name"].lower() == normalized or tutor["display_name"].lower() == normalized
        ),
        TUTOR_PROFILES[0],
    )
